### PR TITLE
 Add data to file_urls column in CSV export

### LINKED ISSUE #8922: Add data to file_urls column in CSV
To aid in the transition to the new `file_urls` column, we should mirror the data in the `file_key` column into the `file_urls` column.

### PR DESCRIPTION
**Pull request by avaleske**
### Description

Mirror data from the `file_key` column to the `file_urls` column in the CSV, to prepare for multi-file upload.

Also
- Updates the multiple file upload column format to be easier to parse. The new column format is a comma separated list, formatted like `civiform.mycity.gov/file1, civiform.mycity.gov/file2`
- Removes the `file_key` column when multiple file upload is enabled, to prevent anyone using a column with incomplete behavior. This mirrors the behavior in the JSON export.

## Release notes

To prepare for multi-file-upload, users of the CSV should transition to using the new `file_urls` column in the CSV. It currently mirrors the data in `file_key` but in the new format.

Once the MULTIPLE_FILE_UPLOAD_ENABLED flag is turned on, the `file_key` column will be removed, so users of the CSV should transition to this column now.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Removed the release notes section if the title is sufficient for the release notes description, or put more details in that section.
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).

### Issue(s) this completes

Fixes #8922

### CODE REVIEW

**Comment by avaleske** [comment]
* **#8884** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8884?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @avaleske and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java [hunk 1]
```diff
@@ -609,10 +609,9 @@ private ImmutableMap<Path, String> getScalarAnswers(ApplicantQuestion question)
                                       programDefinition.id(),
                                       URLEncoder.encode(fileKey, StandardCharsets.UTF_8))
                                   .url())
-                  .collect(Collectors.joining(", ", "[", "]")));
+                  .collect(Collectors.joining(", ")));
```
Comment (devin-civi) [inline server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java:612]: What's the reason for this change (removing the brackets). I think other lists that we export in CSV use the bracket notation so I thought it'd be good to be consistent.

**Review by devin-civi — COMMENTED** [review]
Thanks!

File: server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java (hunk 1 above)
Comment (avaleske) [reply server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java:612]: I was chatting with Matthew in Seattle about the format that would be easiest to parse, and removing the brackets lets you do text-to-columns.

The other thing that uses brackets in this method, the checkbox questions, isn't actually exported that way, and is instead exported as a column for each option.

Comment (devin-civi) [reply server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java:612]: Ah, I didn't realize that. The links also end up formatting properly if you do it this way, which is good.

**Review by devin-civi — APPROVED** [review]

**Review by devin-civi — APPROVED** [review]
